backend/parameter_store.py

```diff
@@ -21,15 +21,15 @@ def _generate_next_value_(  # type: ignore[misc,override] # pylint:disable=no-se
     ) -> str:
         return f"/{ENV}/{name.lower()}"
 
-    DATASETS_TABLE_NAME = auto()
-    DATASET_VERSION_CREATION_STEP_FUNCTION_ARN = auto()
-    IMPORT_ASSET_FILE_FUNCTION_TASK_ARN = auto()
-    IMPORT_METADATA_FILE_FUNCTION_TASK_ARN = auto()
-    IMPORT_DATASET_ROLE_ARN = auto()
+    STORAGE_DATASETS_TABLE_NAME = auto()
+    PROCESSING_DATASET_VERSION_CREATION_STEP_FUNCTION_ARN = auto()
+    PROCESSING_IMPORT_ASSET_FILE_FUNCTION_TASK_ARN = auto()
+    PROCESSING_IMPORT_METADATA_FILE_FUNCTION_TASK_ARN = auto()
+    PROCESSING_IMPORT_DATASET_ROLE_ARN = auto()
     PROCESSING_ASSETS_TABLE_NAME = auto()
     STAGING_BUCKET_NAME = auto()
     STORAGE_BUCKET_NAME = auto()
-    VALIDATION_RESULTS_TABLE_NAME = auto()
```

Comment:
Why is it helpful to know which stack a parameter belongs to?

Reply:
Knowing where parameter belongs is not very important. The reason of adding those stack name prefixes is that we need to make sure that parameter name is changed if it moves between stacks to avoid deployment issues (avoid same parameter name conflict).